README.md

```diff
@@ -225,7 +226,7 @@ You can remove it by running `npm run not-needed — typingsPackageName asOfVers
 
 Any other packages in Definitely Typed that referenced the deleted package should be updated to reference the bundled types.
 You can get this list by looking at the errors from `npm run test-all`.
-To fix the errors, add a `package.json` with `"dependencies": { "foo": "x.y.z" }`.
+To fix the errors, [add a `package.json`](#packagejson) with `"dependencies": { "foo": "x.y.z" }`.
```

Comment:
Should the example and notes below be moved to the package.json section?

Reply:
I thought about that and am happy to go either way. My thoughts were
1. I decided it belonged with "Removing a package". The steps in that section are less followable without it and it doesn't contribute much to the new "package.json" section. Both sections explain `allowedPackageJsonDependencies.txt`.
2. Not having a strong indication in favor of editing it, I opted to minimize this change. Those sections can be reorganized or combined subsequently, and promoting the FAQ entry is neutral at worst, I think?

Reply:
I can live with that.  My thinking was mostly that I would look for package.json info in the corresponding section (which is not the case for an FAQ entry) and might be annoyed to discover that it was elsewhere.  However, as you point out, the sections seem to have different goals and the example makes sense here.